src/PIL/MicImagePlugin.py

```diff
@@ -88,7 +88,9 @@ def seek(self, frame):
         except IndexError:
             raise EOFError("no such frame")
 
+        prev_fp = self.fp
```

Comment:
The `__init__` method does the following:

```py
self.__fp = self.fp
...
self.seek(0)
```

So, this will always result in `self.__fp` being closed immediately. At that point, is the `self.__fp` attribute useful?

---

Why not just close the file on this line instead of using the `prev_fp` temporary variable?

Reply:
I don't think `__fp` is used even in the current master. I imagine that if we ever get a Mic test file with multiple frames, it will become useful, and code would change.

---

Closing the file pointer before `openstream` results in `ValueError: seek of closed file`.